src/Silverback.Integration.Kafka/Messaging/Configuration/KafkaBrokerHeadersEnricher.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using System;
+using Silverback.Messaging.Broker;
+using Silverback.Messaging.Messages;
+using Silverback.Messaging.Outbound.Enrichers;
+
+namespace Silverback.Messaging.Configuration
+{
+    internal sealed class KafkaBrokerHeadersEnricher
```

Comment:
The name should be like `KafkaMovePolicyMessageEnricher` and it should be in `Silverback.Messaging.Outbound.Enrichers` namespace.

Reply:
Fixed, thanks.